app_legacy/Helpers/render/layout.php

```diff
@@ -48,7 +48,7 @@ function RenderToolbar(): void
                 ["systemName" => "SNES/Super Famicom", "listID" => 3],
                 ["systemName" => "Nintendo 64", "listID" => 2],
                 ["systemName" => "Nintendo DS", "listID" => 18],
-                // ["systemName" => "Nintendo DSi", "listID"=>78, "iconName" => "Nintendo DSi"],
+                ["systemName" => "Nintendo DSi", "listID"=>78, "iconName" => "Nintendo DSi"],
```

Comment:
Heads up: the linter is complaining about the missing spaces here - hence the failed action 🖖 
`"listID" => 78,`